src/utils/logger.ts

```diff
@@ -0,0 +1,79 @@
+import { createLogger, transports, format } from "winston";
+
+interface LogLevels {
+  [key: string]: number;
+}
+
+const logLevels: LogLevels = {
+  fatal: 0,
+  error: 1,
+  warn: 2,
+  info: 3,
+  debug: 4,
+  trace: 5,
+};
+
+interface LogLevelColors {
+  [key: string]: string;
+}
+
+const colorizeLevel = (level: string, colors: LogLevelColors) => {
+  switch (level) {
```

Comment:
This switch statement is not really needed.

If the level is not found return default color:

```javascript
const colorizeLevel = (level: string, colors: LogLevelColors) => {
  return colors[level] ?? '\x1b[0m'
}
```